### DISCUSSION TITLE
Help needed.. error: can't find package name in packagelist: !distcc

### DISCUSSION THREAD

**Discussion by planetes1973** [Morganamilo/paru#741] [Q&A]
Hi Everyone. Not sure where to ask this but I'm getting the following message whenever I try to update AUR packages with paru or yay.

error: can't find package name in packagelist: !distcc

Downloading a package with git and running makepkg works fine. This seems to be an issue with the helpers like yay and paru.

I've been getting this message for a month or so and it's finally annoyed me enough to post about it. I've been making packages manually since it started.

It feels like a path or something is screwed up.

paru worked fine for months prior to this starting.

**Comment by Morganamilo** [comment]
Use issues for bugs. And follow the template.

**Reply by planetes1973** [reply]
Ok will do.. new to this so figuring out as I go :)
